packages/react-component-library/src/components/Timeline/Timeline.tsx

```diff
@@ -86,11 +87,20 @@ function getHoursBlockSize(
   )
 }
 
+function getEndDate(startDate: Date, endDate: Date) {
+  if (startDate && endDate && isAfter(startDate, endDate)) {
+    console.error('`startDate` is after `endDate`')
```

Comment:
Can we use the wrapped logger here?

 https://github.com/defencedigital/mod-uk-design-system/blob/master/packages/react-component-library/src/utils/logger.ts